vraptor-core/src/main/java/br/com/caelum/vraptor/interceptor/ExceptionHandlerInterceptor.java

```diff
@@ -93,7 +93,7 @@ protected boolean replay(Exception e) {
 
 		reportException(e);
 
-		logger.debug("handling exception {}", e.getClass());
+		logger.debug("handling exception {}", e);
```

Comment:
maybe you should do `logger.debug("handling exception {}", e.getClass(), e);`, which would print the stacktrace after "Handling exception BazingaException".